.github/workflows/dynamoBinDiff.yml

```diff
@@ -160,15 +160,27 @@ jobs:
         path: |
           ${{ github.workspace }}\net60_Win_Dynamo\bin\NET60_Windows\Release
           ${{ github.workspace }}\net60_Win_Dynamo\.github\scripts
-        key: cache-net60Win-current-${{ github.run_id }}-${{ github.run_attempt }}
+        key: ${{ github.run_id }}-${{ github.run_attempt }}-cache-net60Win-current
     - name: Restore Master Net60 Windows Build
       uses: actions/cache/restore@v3
       with:
         fail-on-cache-miss: true
         path: ${{ github.workspace }}\master_net60_Win_Dynamo\bin\NET60_Windows\Release
-        key: cache-net60Win-master-${{ github.run_id }}-${{ github.run_attempt }}
+        key: ${{ github.run_id }}-${{ github.run_attempt }}-cache-net60Win-master
     - name: Run Binary Diff Job
       run: |
           echo "***Running the binary diff job between the current branch and the master branch of NET60-Windows Config!***"
           cd "$Env:GITHUB_WORKSPACE\net60_Win_Dynamo\.github\scripts"
-          .\bin_diff.ps1 $Env:GITHUB_WORKSPACE\master_net60_Win_Dynamo\bin\NET60_Windows\Release,$Env:GITHUB_WORKSPACE\net60_Win_Dynamo\bin\NET60_Windows\Release
\ No newline at end of file
+          .\bin_diff.ps1 $Env:GITHUB_WORKSPACE\master_net60_Win_Dynamo\bin\NET60_Windows\Release,$Env:GITHUB_WORKSPACE\net60_Win_Dynamo\bin\NET60_Windows\Release -src ${{ github.job }}
+ save-diff-results:
+  needs: [run-bin-diff-net60-windows, run-bin-diff]
+  runs-on: ubuntu-latest
+  steps:
+    - name: Save Diff Results
```

Comment:
Can you elaborate how are the results saved, how long they are preserved, in what format?

Reply:
It does not really save the results but just the PR number to a file, then we upload it to storage and then use it in another workflow delete caches, because PR number is not available in that workflow context.

Reply:
Make sense